src/app/portlets/dot-templates/dot-template-list/dot-template-list.component.ts

```diff
@@ -133,6 +135,10 @@ export class DotTemplateListComponent implements OnInit, OnDestroy {
         return options;
     }
 
+    setContextMenu(template: DotTemplate): void {
+        this.listing.contextMenuItems = this.setTemplateActions(template).map((x) => x.menuItem);
```

Comment:
why this is not going directly in the template?

Reply:
because is a menu that is set in the `p-table` just once, so it changes depending of the row clicked.